Update CSS token per new design system theming capability

fajar-apri-alaska opened the pull request:
# Alaska Airlines Pull Request

**Fixes:** #184 

## Summary:

- Update the design tokens to reflect the new theming conventions regarding tokens from the AuroDesignTokens and JetstreamDesignTokens repo.


## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability


## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

fajar-apri-alaska committed:
perf(token): update token per new theming #184

Changes to be committed:
	modified:   demo/index.html
	modified:   package-lock.json
	modified:   package.json
	modified:   src/style-base.scss
	modified:   src/style-menuoption.scss

blackfalcon commented on package.json:
```diff
@@ -32,8 +32,9 @@
   "devDependencies": {
     "@alaskaairux/design-tokens": "^3.13.0",
     "@alaskaairux/icons": "^4.27.0",
+    "@aurodesignsystem/design-tokens": "^4.1.1",
```
Remove all references to the legacy version of design tokens. Also be sure to update `peerDependencies` too.

blackfalcon commented on package.json:
```diff
@@ -32,8 +32,9 @@
+    "@aurodesignsystem/design-tokens": "^4.1.1",
     "@aurodesignsystem/eslint-config": "^1.3.0",
-    "@aurodesignsystem/webcorestylesheets": "^4.5.0",
+    "@aurodesignsystem/webcorestylesheets": "^5.0.3",
```
Please update `peerDependencies` to match the version update.

fajar-apri-alaska committed:
style: add fallback values to provided tokens

Changes to be committed:
	modified:   src/style-base.scss
	modified:   src/style-menuoption.scss

jason-capsule42 committed:
Merge branch 'main' into fajarapri/updateCSSToken

blackfalcon commented:
:tada: This PR is included in version 3.10.2 :tada:

The release is available on:
- [npm package (@latest dist-tag)](https://www.npmjs.com/package/@aurodesignsystem/auro-menu/v/3.10.2)
- [GitHub release](https://github.com/AlaskaAirlines/auro-menu/releases/tag/v3.10.2)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
